assessments/guide/outline.md

```diff
@@ -0,0 +1,114 @@
+
+    Flibble can be used in any cloud environment.  Three diverse examples are
+    as follows:
+    1. when a Flibble server is used by legacy servers as a
+    database for salted password hashes.
+    2. a Flibble cloudlet may be run
+    on virtualized fog hardware near smartphone users.
+    3. a Flibble distributed service may serve as a backend for a Notary image registry.)
+* Operation.  A description of the operational aspects of the system, such as
+how keys are likely to be managed and stored.
+
+## Project design
+* Design.  A description of the system design that discusses how it works.
+This is especially critical for any mechanisms that relate to the security
+of the system.
+
+## Security analysis
+* Attacker motivations.  A discussion about the likely goals of an attacker.
+This likely relates closely to the impact of different attacks in the
+scenarios.  (e.g., In the password hash case, the attacker wants to expose
+those hashes on the Flibble server.  However, a Flibble cloudlet attacker
+may find it more interesting to bring down the service.)
+* Expected attacker capabilities.  A description of likely capabilities that
+the attacker has in these scenarios should be described.  Both assumptions
+about the strength and limitations of attackers should be described.
+(e.g., We assume that an attacker may be able to exploit implementation
+errors in some set of the servers to take control of them.  However, we
+assume the attacker cannot break AES or SHA256.)
+* Attack risks and effects.  A rough estimation of the risk posed by different attacks, and potential negative consequences.
+(e.g., The master Flibble server only communicates with Flibble servers
+using a minimalistic API that is formally verified and written in Rust.)
+* Security degradation.  A discussion about the resulting security when
+various attacks are launched.  Note, that no system is secure in all
+scenarios, hence it is expected that this will include areas where attacks
+compromise all meaningful security.  (e.g., If an attacker is able to
+compromise the “master” Flibble server, they may read, write, or delete any
+content stored on any system)
+
+## Secure development practices
+* Development pipeline.  A description of the testing and review processes
+that the software undergoes as it is developed and built.
+* Communication Channels. Reference where you document how to reach your
+team or describe in corresponding section.
+  * Internal. How do team members communicate with each other?
+  * Inbound. How do users or prospective users communicate with the team?
+  * Outbound. How do you communicate with your users? (e.g. flibble-announce@ mailing list)
+* Vulnerability Response Process. Who is responsible for responding to a report. What is the reporting process? How would you respond?
```

Comment:
at a minimum shouldn't this SIG be informed "formally"?  i suggest there is a mechanism (GHI, commit to some markdown file, etc) for explicit capture of all known vulns over time.